lib/internal/Magento/Framework/Data/Form/Element/Editor.php

```diff
@@ -310,7 +310,9 @@ protected function _getPluginButtonsHtml($visible = true)
                         if (isset($buttonOptions['style'])) {
                             $configStyle = $buttonOptions['style'];
                         }
+                        // phpcs:disable Magento2.Performance.ForeachArrayMerge
                         $buttonOptions = array_merge($buttonOptions, ['style' => 'display:none;' . $configStyle]);
```

Comment:
This could be reworked into `$buttonOptions['style'] = 'display:none;' . $configStyle;` so we don't need to disable any checks, right?